builtin/logical/pki/storage.go

```diff
@@ -52,6 +57,20 @@ type issuerConfig struct {
 	DefaultIssuerId issuerId `json:"default" structs:"default" mapstructure:"default"`
 }
 
+func genKeyIssuerUUID() string {
+	newId, err := uuid.GenerateUUID()
+	if err != nil {
+		panic("uuid generation failed unexpectedly: " + err.Error())
```

Comment:
Seems a little harsh to panic here, even if it dirties up the function sig.

Reply:
I should contribute that to `go-uuid`. But if `io.ReadFull` fails, your entropy source is completely broken and you should really just panic. There's no cryptographic operations you want to do with a busted entropy source.